com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/Constants.java

```diff
@@ -14,4 +14,5 @@
 public final class Constants {
     public static final String PROJECTNAME = "projectName";
     public static final String DEBUGGEE_ENCODING = "debuggeeEncoding";
+    public static final String MAINCLASS = "mainClass";
```

Comment:
```java
MAIN_CLASS
```

Reply:
Done  with the same issue fixed for project name